datadog/resource_datadog_synthetics_test_.go

```diff
@@ -273,6 +274,9 @@ func syntheticsTestRequestClientCertificateItem() *schema.Schema {
 					Type:        schema.TypeString,
 					Required:    true,
 					Sensitive:   true,
+					StateFunc: func(val interface{}) string {
+						return convertToSha256(val.(string))
+					},
```

Comment:
Won't this make terraform hash the value twice ? once in the read function when you use `getCertificateStateValue`, and once by terraform automatically when you set the value in the state ?

Reply:
Well actually, from my testing, it seems this doesn't do anything at all, let's just remove it and manually set the hashed value directly.

Reply:
No this function never receives a hashed value.